wiremock/bookingStubs.ts

```diff
@@ -0,0 +1,91 @@
+import { stubFor } from './index'
+import bookingDtoFactory from '../server/testutils/factories/bookingDto'
+
+const getCombinations = (arr: Array<string>) => {
+  const result: Array<Array<string>> = []
+  const f = (prefixes: Array<string>, suffixes: Array<string>) => {
```

Comment:
It might be overkill but I would like to see some unit tests on this. I trust it works as expected but would be good for documenation purposes

Reply:
Nice - I got lazy because it was for internal purposes, but writing a test would've helped me a lot when figuring out how to do this!